### DISCUSSION TITLE
Add new Features

### DISCUSSION THREAD

**Discussion by Jonak-Adipta-Kalita** [Jonak-Adipta-Kalita/JAK-Programming-Language#102] [Ideas]
### ✅ Completed:
* `null`, `float` Literal
* Updating Variables without using `var`: `mut`
* Escape Characters for `\n`, `\t`, `\r`, `\"`, `\\`
* `foreach` loop
* `switch`, `elif` expressions
* Add Macros
* `format`, `reverse` function
* File IO
* Add Object Methods

### ⏳ Working on:
* Infix operators: `+=`, `-=`, `*=`, `/=`

### 📒 Currently planned features:
* Package Manager with `JAK-Command-Line-Tool`
* `try` - `except` blocks
* Error Handling Keywords
* Better Error message
* Standard Library

**Comment by mrredo** [comment]
👍

**Comment by Jonak-Adipta-Kalita** [comment]
This Discussion will be used for any future planned features!
